test/test_transforms_tensor.py

```diff
@@ -728,6 +585,132 @@ def test_center_crop(device):
+    for transformed_tensor, transformed_pil_img in zip(transformed_t_list, transformed_p_list):
+        _assert_equal_tensor_to_pil(transformed_tensor, transformed_pil_img)
+
+    transformed_t_list_script = scripted_fn(tensor.detach().clone(), **fn_kwargs)
+    assert len(transformed_t_list) == len(transformed_t_list_script)
+    assert len(transformed_t_list_script) == out_length
+    for transformed_tensor, transformed_tensor_script in zip(transformed_t_list, transformed_t_list_script):
+        assert_equal(transformed_tensor, transformed_tensor_script)
+
+    # test for class interface
+    fn = getattr(T, method)(**meth_kwargs)
+    scripted_fn = torch.jit.script(fn)
+    output = scripted_fn(tensor)
+    assert len(output) == len(transformed_t_list_script)
+
+    # test on batch of tensors
+    batch_tensors = _create_data_batch(height=23, width=34, channels=3, num_samples=4, device=device)
+    torch.manual_seed(12)
+    transformed_batch_list = fn(batch_tensors)
+
+    for i in range(len(batch_tensors)):
+        img_tensor = batch_tensors[i, ...]
+        torch.manual_seed(12)
+        transformed_img_list = fn(img_tensor)
+        for transformed_img, transformed_batch in zip(transformed_img_list, transformed_batch_list):
+            assert_equal(transformed_img, transformed_batch[i, ...])
+
+
+@cpu_only
+@pytest.mark.parametrize('method', ["FiveCrop", "TenCrop"])
+def test_x_crop_save(method):
+    fn = getattr(T, method)(size=[5, ])
+    scripted_fn = torch.jit.script(fn)
+    with get_tmp_dir() as tmp_dir:
+        scripted_fn.save(os.path.join(tmp_dir, "t_op_list_{}.pt".format(method)))
+
+
+@cpu_only
+@pytest.mark.parametrize('size', [32, 34, 35, 36, 38])
+def test_resize_int(size):
+    # TODO: Minimal check for bug-fix, improve this later
+    x = torch.rand(3, 32, 46)
+    t = T.Resize(size=size)
+    y = t(x)
+    # If size is an int, smaller edge of the image will be matched to this number.
+    # i.e, if height > width, then image will be rescaled to (size * height / width, size).
+    assert isinstance(y, torch.Tensor)
+    assert y.shape[1] == size
+    assert y.shape[2] == int(size * 46 / 32)
+
+
+@pytest.mark.parametrize('device', cpu_and_gpu())
+@pytest.mark.parametrize('dt', [None, torch.float32, torch.float64])
+@pytest.mark.parametrize('size', [[32, ], [32, 32], (32, 32), [34, 35]])
+@pytest.mark.parametrize('max_size', [None, 35, 1000])
+@pytest.mark.parametrize('interpolation', [BILINEAR, BICUBIC, NEAREST])
+def test_resize_scripted(dt, size, max_size, interpolation, device):
+    tensor, _ = _create_data(height=34, width=36, device=device)
+    batch_tensors = torch.randint(0, 256, size=(4, 3, 44, 56), dtype=torch.uint8, device=device)
+
+    if dt is not None:
+        # This is a trivial cast to float of uint8 data to test all cases
+        tensor = tensor.to(dt)
+    if max_size is not None and isinstance(size, Sequence) and len(size) != 1:
+        pytest.xfail()
```

Comment:
let's give a reason in the xfail message:
```suggestion
        pytest.xfail("with max_size, size must be a sequence with 2 elements")
```